src/PrestaShopBundle/ApiPlatform/Resources/Hook.php

```diff
@@ -61,18 +61,21 @@
                     [
                         'name' => 'Authorization',
                         'in' => 'scopes',
-                        'description' => 'read:hook-status <br> write:hook-status ',
+                        'description' => 'hook-status_read <br> hook-status_write ',
```

Comment:
```suggestion
                        'description' => 'hook_read <br> hook_write ',
```

Reply:
The scope are snake case no `-` allowed Besides we don't define sub scopes for sub parts of the entity for now So even for the status endpoint the scope needed should the same as the one for the whole hook `hook_read`